azurelinuxagent/common/protocol/goal_state.py

```diff
@@ -0,0 +1,411 @@
+# Microsoft Azure Linux Agent
```

Comment:
Most of the code here comes from wire.py, but please do a careful review. Thanks!